client/utils/build-challenges.js

```diff
@@ -38,6 +38,13 @@ exports.replaceChallengeNode = () => {
 
 exports.buildChallenges = async function buildChallenges() {
   const curriculum = await getChallengesForLang(curriculumLocale);
+
+  // temp removal of rdbms from production
+  if (deploymentEnv === 'live') {
+    console.log('here');
```

Comment:
```suggestion
  if (deploymentEnv !== 'staging') {
```
Makes it default to 'safe' behaviour.

Reply:
changes added